crates/goose/src/systems/goose_hints.rs

```diff
@@ -69,13 +68,17 @@ impl System for GooseHintsSystem {
         &[]
     }
 
-    async fn status(&self) -> AnyhowResult<HashMap<String, Value>> {
-        Ok(HashMap::new())
+    async fn status(&self) -> AnyhowResult<Vec<Resource>> {
+        Ok(Vec::new())
     }
 
     async fn call(&self, tool_call: ToolCall) -> AgentResult<Vec<Content>> {
         Err(AgentError::ToolNotFound(tool_call.name))
     }
+
+    async fn read_resource(&self, _uri: &str) -> AgentResult<String> {
```

Comment:
should we follow MCP here? the server responds with a list of content objects - https://modelcontextprotocol.io/docs/concepts/resources#reading-resources

Reply:
+1 decided to keep like this since a bit simpler. easy fast follow